next-app/postcss.config.cjs

```diff
@@ -3,4 +3,4 @@ module.exports = {
     tailwindcss: {},
     autoprefixer: {},
   },
-}
+};
```

Comment:
unless this becomes some kind of Holy War, I'd prefer not having the semi-colons.

Reply:
My vote would definitely be **for** semi-colons 😉. I think that utilizing the distinction between optional and required semi-colons primarily just adds confusion.

uh oh 😄